src/sonic-yang-models/yang-models/sonic-device_metadata.yang

```diff
@@ -204,6 +204,14 @@ module sonic-device_metadata {
                     description "Advertise Loopback0 interface IPv6 /128 subnet address as it is with set to true.
                                  By default SONiC advertises /128 subnet prefix in Loopback0 as /64 subnet route";
                 }
+
+                leaf timezone {
+                    type stypes:timezone-name-type {
```

Comment:
Why can't we just have type as string and update the description here? We add in sonic-types if the type is used across multiple files or has a complexity.

Reply:
`stypes:timezone-name-type` is defined by OpenConfig. I want to be consistent with that definition, so I copied it from it.
I prefer to leave it in that format, so once it will be changed by OpenConfig (unlikely but hey 😄), we can change it only in `sonic-types`